test/e2e/e2e_coverage_and_cleanup_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var workerNodeA = "kind-worker"
+var workerNodeB = "kind-worker2"
```

Comment:
Is it possible that theia-manager or clickhouse-monitor is deployed on `kind-control-plane`?

Reply:
no, they'll never be deployed on kind-control-plane